Docs/Book/Cookbook_v2.rst

```diff
@@ -0,0 +1,1017 @@
+RDKit Cookbook v2
+%%%%%%%%%%%%%%%%%%%%
+
+.. sectionauthor:: Vincent F. Scalfani <vfscalfani@ua.edu>
+
+Introduction
+**************
+
+What is this?
+===============
+
+This document provides example recipes of how to carry out particular tasks using the RDKit 
+functionality from Python. The contents have been contributed by the RDKit community, 
+tested with the latest RDKit release, and then compiled into this document. Note that this 
+document is the second iteration of the :doc:`Cookbook` (i.e., v2). The old Cookbook written 
+in Markdown is no longer maintained, but is available in prior RDKit releases for reference. 
+The RDKit Cookbook v2 is written in reStructuredText, which supports Sphinx doctests, 
+allowing for easier validation and maintenance of the RDKit Cookbook v2 code examples, where appropriate. 
+
+What gets included?
+=====================
+
+The examples included come from various online sources such as blogs, shared gists, and 
+the RDKit mailing lists. Generally, only minimal editing is added to the example code/notes for 
+formatting consistency and to incorporate the doctests. We have made a conscious effort 
+to appropriately credit the original source and authors. One of the first priorities of this
+document is to compile useful **short** examples shared on the RDKit mailing lists, as 
+these can be difficult to discover. It will take some time, but we hope to expand this 
+document into 100s of examples. As the document grows, it may make sense to prioritize 
+examples included in the RDKit Cookbook v2 based on community demand.
+
+Feedback and Contributing
+==========================
+
+If you have suggestions for how to improve the Cookbook v2 and/or examples you would like 
+included, please contribute directly in the source document (the .rst file).
+Alternatively, you can also send Cookbook revisions and addition requests to the mailing list:
+<rdkit-discuss@lists.sourceforge.net> (you will need to subscribe first).
+
+.. note::
+
+   The Index ID# (e.g., **RDKitCB_##**) is simply a way to track Cookbook entries and image file names. 
+   New Cookbook additions are sequentially index numbered, regardless of where they are placed 
+   within the document. As such, for reference, the next Cookbook entry is **RDKitCB_20**.
+
+Drawing Molecules (in a Jupyter Environment)
+**********************************************
+
+Include an Atom Index
+======================
+
+| **Author:** Takayuki Serizawa
+| **Source:** `<https://iwatobipen.wordpress.com/2017/02/25/draw-molecule-with-atom-index-in-rdkit/>`_
+| **Index ID#:** RDKitCB_0
+| **Summary:** Draw a molecule with atom index numbers.
+
+.. testcode::
+
+   from rdkit import Chem
+   from rdkit.Chem.Draw import IPythonConsole
+   from rdkit.Chem import Draw
+   IPythonConsole.ipython_useSVG=False
+   import rdkit
```

Comment:
you don't need the `import rdkit`
I include that in notebooks so that I can do `print(rdkit.__version__)`, but that's not needed here
I guess most of the other appearances of `import rdkit` in this document can also be removed

Reply:
I removed all `import rdkit` from document.

Reply:
hmm, I had done a PR this morning that included that change along with the rename and a few other cleanups/clarifications. did you not see that?

Reply:
@greglandrum  oh, my bad. I didn't get a notification for that, but I see it now. What is easiest way to fix the conflicts?

Reply:
it's actually probably easiest for me to do it. I'll update the PR